tox.ini

```diff
@@ -16,9 +21,13 @@ commands =
 
 [testenv:docs]
 changedir = docs
-whitelist_externals =
+allowlist_externals =
+    poetry
     mkdir
     rm
+commands_pre =
```

Comment:
You don't need to repeat that block though, it's going to be using the one from `testenv` by default.